init/common.js

```diff
@@ -202,6 +204,30 @@ const createProject = async (interestId, userId, project, addTeam=true) => {
   });
 };
 
+const createOpportunities = async (interestId) => {
+
+  for (let index = 0; index < config.opportunity.length; index++) {
+    try{
+      let existing = await strapi.entityService.findMany("api::opportunity.opportunity", {filters: {title: config.opportunity[index].title}});
+      existing = existing[0];
+
+      if (existing) {
+        return;
+      }
+      
```

Comment:
I have a question. Here do you want to stop just the current iteration. Or the entire loop ?
And why ?